source/deployment/guest-accounts.rst

```diff
@@ -3,63 +3,105 @@
+
+
+Inviting Guests to the Mattermost Server via Email
 ------------------------------------------------------
-Guests can be invited into one for more Mattermost channels within a team by System Administrators and roles that have the **Invite Guest** permission. A guest can also be invited into channels on multiple teams. 
 
-**Note:** Guest invitations will be revoked after 48 hours per the member email invitation process. If your guest has not accepted the invitation within that period, please follow the steps below to resend an invitation to the guest again. 
+Guests can be invited into one for more Mattermost channels within a team by System Administrators and roles that have the **Invite Guest** permission. A guest can also be invited into channels on multiple teams.
+
+**Note:** Guest invitations will be revoked after 48 hours per the member email invitation process. If your guest has not accepted the invitation within that period, please follow the steps below to resend an invitation to the guest.
 
 To invite guests into one or more Mattermost channels:
 
-1. Navigate to **Main Menu > Invite People**.  This is a new invitation option that combines inviting guests and members and consolidates **Send Email**, **Get Team Invite Link**, and **Add Members to the Team**. 
-2. Select **Invite Guests**.
-3. Enter the guest’s email address.
-4. Choose the channels the guest can join. 
-5. (Optional) Enter a custom message.
+1. Enable email invites from **System Console > Signup > Enable Email Invitations**.
+2. Navigate to **Main Menu > Invite People**.  This is a new invitation option that combines inviting guests and members and consolidates **Send Email**, **Get Team Invite Link**, and **Add Members to the Team**.
+3. Select **Invite Guests**.
+4. Enter the guest’s email address.
+5. Choose the channels the guest can join (excluding managed teams).
+6. (Optional) Enter a custom message.
 
 .. image:: ../images/Guest_Invite_Screen.png
 
-Guest Permission Settings 
+Configuring AD/LDAP Authentication
+----------------------------------
+
+When enabled, the **Guest Filter** in Mattermost identifies external users whose AD/LDAP role is guest and who are invited to join 
+your Mattermost server. These users will have the Guest role applied immediately upon first sign-in instead of the default member user role.
+This eliminates having to manually assign the role in the System Console.
+
+1. Enable Guest Access via **System Console > Guest Access (Beta)**.
+2. Navigate to **System Console > Authentication > AD/LDAP**.
+3. Complete the **Guest Filter** field.
+4. Choose **Save**.
+
+If a Mattermost Guest user has the guest role removed in the AD/LDAP system, the synchronization process
+will not automatically promote them to a member user role. This is done manually via **System Console > User Management**. If a
+member user has the **Guest Attribute** added, the synchronization processes will automatically demote the member user to the guest role.
+
```

Comment:
Worth noting that a Guest will be taken to a generic screen letting them know they have no channels assigned until they have been added by a role with invite guest permissions.